### PR TITLE
Handle better jinja2 exceptions in Handler

### PR DESCRIPTION
**Pull request by mithunbharadwaj**
1. Updated changeling
2. Added more exceptions in the handler related to jinja2
3. Updated a nice-looking library template for testing.

### CODE REVIEW

File: asabiris/handlers/webhandler.py [hunk 1]
```diff
@@ -147,9 +147,16 @@ async def send_slack(self, request, *, json_data):
 
 		try:
 			await self.App.SendSlackOrchestrator.send_to_slack(json_data)
-
 		except jinja2.exceptions.UndefinedError as e:
```
Comment (eliska-n) [inline asabiris/handlers/webhandler.py:150]: you should catch this error where you apply jinja params. 
when sending the error message on BE, provide at least the file name (location in the library) that is not right. Tell the user what they should do to fix the situation. I.e. PathError means the template is not in the proper place of the Library, right? Provide some hints in the response how to fix the problem.

**Review by eliska-n — CHANGES_REQUESTED** [review]

**Comment by mithunbharadwaj** [comment]
@eliska-n Respone

```
{
    "result": "ERROR",
    "message": "''tenant' is undefined' in Jinja2 template '/Templates/Slack/message.md'.",
    "uuid": "aaca2d9d-0347-4c9d-abab-011e45c65769"
}
```

**Comment by eliska-n** [comment]
Can you show here how the error looks like when you use template from /Templates/Email instead of /Templates/Slack?

**Comment by mithunbharadwaj** [comment]
@eliska-n 


```
{
    "result": "NOT-FOUND",
    "message": "The entered path '/Templates/Email/message.md' is not correct. Please move your files to '/Templates/Slack/'.",
    "uuid": "0bb5a79e-f60e-48c6-a190-48bbcd8dcb54"
}
```

File: asabiris/exceptions.py [hunk 2]
```diff
@@ -35,3 +35,17 @@ def __init__(self, message=None, *args, format=None):
...
+	def __init__(self, message=None, *args, template_path=None, variable_name=None):
+		self.TemplatePath = template_path
+		self.VariableName = variable_name
+
+		if message is not None:
+			super().__init__(message, *args)
+		elif template_path is not None and variable_name is not None:
+			message = "'{}' in Jinja2 template '{}'.".format(variable_name, template_path)
+			super().__init__(message, *args)
+		else:
```
Comment (eliska-n) [inline asabiris/exceptions.py:50]: this is the same as the code on L46 - there are just 2 branches? if and else

**Review by eliska-n — CHANGES_REQUESTED** [review]

**Pull request merged**